_inc/client/components/jetpack-notices/index.jsx

```diff
@@ -162,20 +163,15 @@ export const UserUnlinked = React.createClass( {
 			this.props.connectUrl &&
 			this.props.siteConnected
 		) {
-			const text = __( 'You are not connected to WordPress.com.' );
-
 			return (
-				<SimpleNotice
-					showDismiss={ false }
-					status="is-info"
-					text={ text }
-				>
-					<NoticeAction
+				<div className="jp-unlinked-notice">
+					<JetpackBanner
+						title={ __( 'Connect your account to get the most out of Jetpack' ) }
+						callToAction={ __( 'Connect to WordPress.com' ) }
 						href={ `${ this.props.connectUrl }&from=unlinked-user-connect` }
-					>
-						{ __( 'Link to WordPress.com' ) }
-					</NoticeAction>
-				</SimpleNotice>
+						icon="my-sites"
+					/>
```

Comment:
Has the same duplicated icon issue
https://github.com/Automattic/jetpack/pull/7606/files#r131984002
But let's dismiss it on this PR and fix on the other.